include/nuttx/video/fb.h

```diff
@@ -47,183 +47,183 @@
-                                           *         boundaries */
+                                           *         boundaries              */
 #define FB_FMT_IMC3           56          /* BPP=12  As IMC1 except that UV
-                                           *         swapped */
+                                           *         swapped                 */
 #define FB_FMT_IMC4           57          /* BPP=12  As IMC2  except that UV
-                                           *         swapped */
+                                           *         swapped                 */
 #define FB_FMT_CLPL           58          /* BPP=12  YV12 but including a level
-                                           *         of indirection. */
-#define FB_FMT_Y41B           59          /* BPP=12?  4:1:1 planar. */
-#define FB_FMT_Y42B           60          /* BPP=16?  YUV 4:2:2 planar. */
-#define FB_FMT_CXY1           61          /* BPP=12 */
-#define FB_FMT_CXY2           62          /* BPP=16 */
+                                           *         of indirection.         */
+#define FB_FMT_Y41B           59          /* BPP=12?  4:1:1 planar.          */
+#define FB_FMT_Y42B           60          /* BPP=16?  YUV 4:2:2 planar.      */
+#define FB_FMT_CXY1           61          /* BPP=12                          */
+#define FB_FMT_CXY2           62          /* BPP=16                          */
 
 #define FB_ISYUVPLANAR(f)     (((f) >= FB_FMT_YVU9) && (f) <= FB_FMT_CXY2)
 #define FB_ISYUV(f)           (FB_ISYUVPACKED(f) || FB_ISYUVPLANAR(f))
 
 /* Hardware cursor control **************************************************/
 
 #ifdef CONFIG_FB_HWCURSOR
-#  define FB_CUR_ENABLE       0x01        /* Enable the cursor */
-#  define FB_CUR_SETIMAGE     0x02        /* Set the cursor image */
-#  define FB_CUR_SETPOSITION  0x04        /* Set the position of the cursor */
-#  define FB_CUR_SETSIZE      0x08        /* Set the size of the cursor */
-#  define FB_CUR_XOR          0x10        /* Use XOR vs COPY ROP on image */
+#  define FB_CUR_ENABLE       0x01      /* Enable the cursor                 */
+#  define FB_CUR_SETIMAGE     0x02      /* Set the cursor image              */
+#  define FB_CUR_SETPOSITION  0x04      /* Set the position of the cursor    */
+#  define FB_CUR_SETSIZE      0x08      /* Set the size of the cursor        */
+#  define FB_CUR_XOR          0x10      /* Use XOR vs COPY ROP on image      */
 #endif
 
 /* Hardware overlay acceleration ********************************************/
 
 #define FB_NO_OVERLAY         -1
 
 #ifdef CONFIG_FB_OVERLAY
-#  define FB_ACCL_TRANSP      0x01        /* Hardware tranparency support */
-#  define FB_ACCL_CHROMA      0x02        /* Hardware chromakey support */
-#  define FB_ACCL_COLOR       0x04        /* Hardware color support */
-#  define FB_ACCL_AREA        0x08        /* Hardware support area selection */
+#  define FB_ACCL_TRANSP      0x01      /* Hardware tranparency support      */
```

Comment:
could you move the style change to new patch?

Reply:
Interesting point - I think the style should be consistent. So do I style my code to match the old, or change the old to match the new???

Reply:
either way is fine if nxstyle doesn't generate the warning, but please split to new patch, don't mix the real change and style change in one patch. It make the reviewer hard to find the real change.

Reply:
Ok - certainly don't want to make your task harder than it is. I appreciate the explanation :-)

I will go through and undo style changes from the original code - but won't be until Monday

Reply:
I checked style guide:
>Line Width. Text should not extend past column 78 in the typical C source or header file. Sometimes the nature of the content of a file may require that the lines exceed this limit. This often occurs in header files with naturally long definitions. If the line width must extend 78 lines, then some wider line width may be used in the file provided that it is used consistently.

So, as you say, long lines are OK in header files if consistent, so will change my "behaviour" now :-)

Reply:
nxstyle doesn't check the macro length, but anyway it's fine to reduce the length, but please move the style change to new patch.